tasks/Module 2/task_1.py

```diff
@@ -0,0 +1,55 @@
+from collections import defaultdict
+
+ITEMS_TO_SKIP = ['chewed gum', 'rubbish', 'used tissue']
+MAX_INVENTORY_CAP = 80
+HEAVY_WEIGHT_THRESHOLD = 70
+LIGHT_WEIGHT_THRESHOLD = 60
+
+
+def display_inventory(inventory):
+    '''
+    Displays the contents of the inventory
+    '''
+    print("Inventory:")
```

Comment:
I'd move this print strictly over the for loop on line 15. Good to keep it as close to each other as possible